ledger-core/v2/virtual/handlers/vdelegatedrequestfinished.go

```diff
@@ -86,25 +113,27 @@ func (s *SMVDelegatedRequestFinished) stepProcess(ctx smachine.ExecutionContext)
 		case payload.CallIntolerable:
 			state.ActiveImmutablePendingCount--
```

Comment:
this one can underflow too

and, probably, it would be nice to log events that are not expeected, somethin like unexpected Pending finished for object %ref%. In case of error it would help

Reply:
@sexyhamster Fixed